README.md

```diff
@@ -138,9 +138,9 @@ $faker->ext(PersonDe::class)->firstName(); // German name
 use Faker\Factory;
 
 $faker = Factory::create();
-$faker->withUnique()->name();
-$faker->withMaybe(0.8)->name();
-$faker->withValid(fn($v) => strlen($v) > 3))->name();
+$faker->unique()->name();
+$faker->maybe(0.8)->name();
+$faker->satisfies(fn($v) => strlen($v) > 3))->name();
```

Comment:
Out of scope of this pull request, but I am not entirely sure yet how to deal with these specifications. 

I think it's probably ok for the user to run into type errors when we do something like 

```php
<?php

$faker->satisfies(fn(string $v) => strlen($v) > 3))->dateOfBirth();
```

Reply:
Hm.. It donsnt always makes sense... True. 

But this is just the same as the current implementation. Are you saying that we maybe should remove this in 2.0?

Reply:
No, I think it's cool to have this!